cmd/create/runner.go

```diff
@@ -276,7 +276,11 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 
 	r.logger.LogCtx(context.Background(), "message", "setup complete")
 
-	// TODO: Store kubeconfig somewhere
+	// Write to path from flag
+	err = ioutil.WriteFile(r.flag.OutputPath, []byte(kubeconfig), 0644)
```

Comment:
Why go though the extra hoop here to create the kubeconfig through calling gsctl with json output flag if its already possible for gsctl to write the kubeconfig by itself?

Reply:
I guess that would also work. I still have some doubts about the Tekton workspace so we might need to put it in a Secret after all (that's why it was done like this originally), but if it works with the Tekton workspace then I'll come back and use gsctl directly. I added a note about this in the issue